### ISSUE TITLE
Notifications feature

### ISSUE THREAD

**Issue by markhallak** [markhallak/AnxietyGo#18]
Create two topics:
- Affirmations
- Mood

**Comment by markhallak** [comment]
Tested it. Not working properly. It is only useful if we want to inform the users about a new update.

I think it is best if we use local notifications.

**Comment by ShivamMenda** [comment]
Hey @markhallak I'll work on this with you 👍

**Comment by ShivamMenda** [comment]
@markhallak I think we can work with Local notifications.

**Comment by markhallak** [comment]
Yeah absolutely. I will look into some libraries and test them so I can pick the best one for the project.

There is a library in mind called "awesome notifications". Have you used it before?

**Comment by vikas-shrma** [comment]
what is the motive of FCM here? can anyone explain in bit details

**Comment by markhallak** [comment]
@vikas-shrma We are still testing things out. I am open to any suggestions.

**Comment by ShivamMenda** [comment]
@markhallak We can use local push notifications package. I've used it before. We can schedule notifications weekly, daily or monthly at a particular time too.

**Comment by ShivamMenda** [comment]
@markhallak Can you keep this issue open so that it doesn't get lost? Keep a separate tag as Not planned also.

**Comment by markhallak** [comment]
@ShivamMenda It seems like it is the way to go. I will test it soon.

I will reopen it.

**Comment by ShivamMenda** [comment]
@markhallak Yeah sure will join you in a few days once exams are done.
